django_netjsongraph/base/admin.py

```diff
@@ -23,6 +23,20 @@ def __init__(self, *args, **kwargs):
 class BaseAdmin(TimeStampedEditableAdmin):
     save_on_top = True
 
+    def get_extra_context(self, pk=None):
```

Comment:
Then the `View topology graph` button will appear in all the change views. Of Node, Link and Topology. Is it okay?